setup.py

```diff
@@ -31,6 +31,7 @@
     maintainer="The RIPE Atlas team",
     maintainer_email="atlas@ripe.net",
     install_requires=[
+        "IPy",
         "python-dateutil",
```

Comment:
Actually, I think Chris just patched Sagan to remove its use in the traceroute model.  You may want to check, but it might be the case that Sagan no longer requires it.